src/Microsoft.AspNetCore.Razor.LanguageServer.Common/AdhocLanguageServices.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Razor;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Common
+{
+    public class AdhocLanguageServices : HostLanguageServices
```

Comment:
Pulled out all of these adhoc services from the LanguageServer's project snapshot manager accessor. Direct Cut & Paste

Reply:
Can't be IVT because they're used in the non-strong named assemblies. Either way they only exist in order t o spin up fake workspaces for the language server (no workspace) and for the OmniSharp plugin tests which need to fake a workspace to act as OmniSharp's workspace.